aca_preview/preview.py

```diff
@@ -184,6 +184,7 @@ def add_review_methods(cls, aca):
 
         """
         aca.__class__ = cls
+        aca.add_row_col()
```

Comment:
This may be controversial.  I mostly just wanted it for the tests, but then I didn't see any downside to guaranteeing that stars had row and column if they got into an ACAReviewTable.

Reply:
Looks good to me